### PR TITLE
MikroTik intercell

### PR DESCRIPTION
**Pull request by gjalves**
Support for Mikrotik Intercell

### CODE REVIEW

File: lte/gateway/python/magma/enodebd/devices/experimental/cavium.py [hunk 1]
```diff
@@ -118,7 +116,7 @@ def __init__(self, acs: EnodebAcsStateMachine, when_done: str):
         self.acs = acs
         self.done_transition = when_done
 
-    def get_msg(self, message: Any) -> AcsMsgAndTransition:
+    def get_msg(self) -> AcsMsgAndTransition:
```
Comment (xjtian) [inline lte/gateway/python/magma/enodebd/devices/experimental/cavium.py:119]: Can you revert the signature change to this function please - we did a minor refactor of the state machine interface recently

File: lte/gateway/python/magma/enodebd/devices/experimental/cavium.py [hunk 2]
```diff
@@ -168,7 +166,7 @@ def read_msg(self, message: Any) -> AcsReadMsgResult:
             return AcsReadMsgResult(False, None)
         return AcsReadMsgResult(True, None)
 
-    def get_msg(self, message: Any) -> AcsMsgAndTransition:
+    def get_msg(self) -> AcsMsgAndTransition:
```
Comment (xjtian) [inline lte/gateway/python/magma/enodebd/devices/experimental/cavium.py:169]: revert this line too please

File: lte/gateway/python/magma/enodebd/devices/experimental/cavium.py [hunk 3]
```diff
@@ -79,7 +77,7 @@ def _init_state_map(self) -> None:
             'wait_post_reboot_inform': WaitInformMRebootState(self, when_done='wait_reboot_delay', when_timeout='wait_inform'),
             # The states below are entered when an unexpected message type is
             # received
-            'unexpected_fault': ErrorState(self, inform_transition_target='wait_inform')
+            'unexpected_fault': ErrorState(self)
```
Comment (xjtian) [inline lte/gateway/python/magma/enodebd/devices/experimental/cavium.py:80]: revert this line please - we added some remediation logic to this ErrorState state

**Review by xjtian — CHANGES_REQUESTED** [review]
Thanks for this @gjalves! Looks good to me, just need to pull in a few changes that we've made to the state machine recently.

**Comment by 119Vik** [comment]
Can one of the admins verify this patch?

**Comment by electronjoe** [comment]
Hello @gjalves! Apologies for the latency on these reviews. I'm trying to help shepherd our oldest PRs to a resolution.

Could you address the several Codacity linting warnings on this PR - regarding e.g. unused imports?

IMO even if Magma is unable to test a PR with hardware, we should be able to assess low risk on accepting contributions like this one. I am happy to help move this forward if you are able.

**Pull request closed without merging**